pkg/users/usersGet_test.go

```diff
@@ -65,56 +71,40 @@ func mockUsersServlet(t *testing.T, writer http.ResponseWriter, request *http.Re
 				"last-login": "2024-10-28T15:32:49.546029Z",
 				"client-name": "rest-api"
 			}
-		]
+		],
+		"synthetic" : {
+			"role": {
+				"metadata" : {
+					"name" : "admin"
+				}
+			}
+		}
     }    
 ]
 `
-	} else if state == "empty" {
```

Comment:
Really surprised to see this was merged in - I commented on using interactions instead of this if-elseif stuff a while ago https://github.com/galasa-dev/cli/pull/304#discussion_r1824305591 but it must've been accidentally ignored :(

Thanks for making these changes!

Reply:
We could create objects using the generated beans and marshal them into JSON strings instead of hard-coding JSON response structures in the tests? Just a thought!